ci/scripts/connect-release-init-npm.js

```diff
@@ -47,6 +47,16 @@ const findIndexByCommit = (commitArr, searchString) =>
     commitArr.findIndex(commit => commit.includes(searchString));
 
 const initConnectRelease = async () => {
+    console.log('Using GitHub Token:', process.env.GITHUB_TOKEN ? 'Yes' : 'No');
+
+    if (process.env.GITHUB_TOKEN) {
+        // Making sure we use the proper GITHUB_TOKEN
+        exec('gh', ['auth', 'setup-git']);
+        exec('gh', ['config', 'set', '-h', 'github.com', 'oauth_token', process.env.GITHUB_TOKEN]);
+    } else {
```

Comment:
It was not required before to do anything like this in the script to be able to use the proper user but maybe since we have made different configurations might be required now?